.packit.yaml

```
@@ -34,6 +35,12 @@ jobs:
           - https://yum.theforeman.org/katello/nightly/katello/el9/x86_64/
     module_hotfixes: true
 
+  - <<: *copr
+    trigger: commit
+    branch: develop
```

This project uses `master` as the branch, so
```suggestion
    branch: master
```